ci/check_style.sh

```diff
@@ -16,7 +16,9 @@ rapids-dependency-file-generator \
 rapids-mamba-retry env create --force -f "${ENV_YAML_DIR}/env.yaml" -n checks
 conda activate checks
 
-FORMAT_FILE_URL=https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-24.04/cmake-format-rapids-cmake.json
+RAPIDS_VERSION_MAJOR_MINOR="$(sed -E -e 's/^([0-9]{2})\.([0-9]{2})\.([0-9]{2})/\1.\2/' VERSION)"
```

Comment:
This assumes the script is invoked from the repository root. Can we make it agnostic to that?

Reply:
`ci/check_style.sh` already assumes it's being run from the repository root due to its usage of `rapids-dependency-file-generator`. Same for `ci/build_docs.sh`.